config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -88,51 +78,8 @@ teams:
     - justaugustus
     - lachie83
     - LappleApple
-    - mattfarina
-    privacy: closed
-  sig-contributor-experience:
```

Comment:
Completely removing this?

Reply:
I need to fix the formatting, but probably? the group is very stale. The instances that I know it where it was used, it was more or less targeted at the leads.

ref: https://github.com/search?o=desc&q=org%3Akubernetes+team%3Akubernetes%2Fsig-contributor-experience&s=created&type=Issues